### DISCUSSION TITLE
GitHub Mobile: Issue: net::ERR_UNKNOWN_URL_SCHEME

### DISCUSSION THREAD

**Discussion by shivagnihotri** [community/community#55847] [Mobile]
### Select Topic Area

Bug

### Body

Greetings, GitHub Mobile developers! :)

First off: Why is there no official repository for GitHub mobile where I could add this issue? Strange thing.

Bug: I cannot use the GitHub mobile app because a login is not possible due to an "unknown URL scheme". :sad

![Unknown URL scheme](https://github.com/community/community/assets/97066277/4327d51b-98eb-465a-be76-fb1bcf7f3a62)

**Comment by Tammynav1** [comment]
[@****]()
